mkosi

```diff
@@ -1531,6 +1648,14 @@ def process_setting(args, section, key, value):
             return True
         else:
             return False
+    elif section == "LUKS":
```

Comment:
I'd not add a new section for this, but stick it in the same one as Verity= and stuff.